### PR TITLE
Add tests

### LINKED ISSUE #186: Tests?
It'd be nice to get some coverage on these homegrown utils with `jest`.

Maybe like so:
```
src
└─ helpers
   ├── query-string.ts
   └── query-string.spec.ts
```

### PR DESCRIPTION
**Pull request by altruisticsoftware**
🚀 

Closes #186

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,18 +19,18 @@ You don't need to do anything other than enable HTTPS on your website. To ensure
...
-
-    var tsml_react_config = {
-      timezone: 'Pacific/Honolulu',
-      strings: {
-        en: {
-          types: {
-            BEACH: 'Beach Meeting',
-          },
-        },
+```js
```
Comment (joshreisner) [inline README.md:22]: nice

**Review by joshreisner — APPROVED** [review]
so exciting

**Comment by joshreisner** [comment]
Screen Shot 2022-02-03 at 4 57 43 PM

so cool. we'll get to 100% test coverage eventually.

**Comment by altruisticsoftware** [comment]
@joshreisner  You're dang right we will! 🔥

**Comment by gobborg** [comment]
@myarete can you please reach out to Renae on C4R Slack?

**Pull request merged**